r2i/tensorrt/engine.cc

```diff
@@ -0,0 +1,168 @@
+/* Copyright (C) 2018-2020 RidgeRun, LLC (http://www.ridgerun.com)
+ * All Rights Reserved.
+ *
+ * The contents of this software are proprietary and confidential to RidgeRun,
+ * LLC.  No part of this program may be photocopied, reproduced or translated
+ * into another programming language without prior written consent of
+ * RidgeRun, LLC.  The user is free to modify the source code after obtaining
+ * a software license from RidgeRun.  All source code changes must be provided
+ * back to RidgeRun without any encumbrance.
+ */
+
+#include "r2i/tensorrt/engine.h"
+#include "r2i/tensorrt/prediction.h"
+#include "r2i/tensorrt/frame.h"
+
+#include <cuda.h>
+#include <cuda_runtime.h>
+#include <NvInfer.h>
+#include "NvInferPlugin.h"
+#include <vector>
+
+static int
+getSizeFromType (nvinfer1::DataType type) {
```

Comment:
I feel this is a duplicate with our new datatype class. Can we convert nvinfer::DataType into r2i::DataType and get the size from there?